var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -209,6 +206,13 @@ def install(self, spec, prefix):
             '--with-debugging=%s' % ('1' if '+debug' in spec else '0'),
             '--with-64-bit-indices=%s' % ('1' if '+int64' in spec else '0')
         ])
+        if '+debug' in spec:
+            options.append('--with-debugging=%s')
```

Comment:
Did you intend to substitute this `%s` with something?

Reply:
bad code. fixed now. Thanks!